src/PrestaShopBundle/Resources/config/services/adapter/product/_combination.yml

```diff
@@ -92,8 +92,8 @@ services:
         arguments:
             - '@prestashop.adapter.product.combination.repository.combination_repository'
 
-    prestashop.adapter.product.combination.update.combination_remover:
```

Comment:
It's a BC Break as this service has been defined in 178x. Could you document it ?

Reply:
:+1: updated in the PR description